golden_response/staging/get_stat_ranking/crime_percapita.json

```diff
@@ -10,1383 +10,1383 @@
           },
           {
             "rank": 2,
+            "value": 60960,
```

Comment:
My guess is this geo might have changed typeOfs. shanth?

Reply:
I'll take a look at this later tonight (filed b/170684975)..

Reply:
It might be easier to debug if we push to staging, to detect other bugs.